test/ice_nucleation_calibration.jl

```diff
@@ -0,0 +1,278 @@
+import EnsembleKalmanProcesses as EKP
```

Comment:
We should add this file to `runtestst.jl` . Otherwise it will not be executed in the GHA tests.